### DISCUSSION TITLE
String shows as unchanged but has been translated and shown in history

### DISCUSSION THREAD

**Discussion by ghubz** [WeblateOrg/weblate#6177] [Q&A]
After the translations for some languages have been added and everything was fine, now out of a sudden the translations done are shown in history and the strings are shown as unchanged! It's like they've been reverted. 

No changes have been made to the system. There are no pending transactions neither in celery or local repo. 

Seems similar to this https://github.com/WeblateOrg/weblate/issues/6058

When viewing the project list "Untranslated", "Untranslated Words" & "Untranslated Characters" are all empty. When entering into the language it shows "Failed check: Unchanged translation — 553 words"

Any ideas what could have happened or how to fix? 

> * Weblate: 4.6.2
>  * Django: 3.2
>  * siphashc: 2.1
>  * translate-toolkit: 3.3.6
>  * lxml: 4.6.3
>  * Pillow: 8.2.0
>  * bleach: 3.3.0
>  * python-dateutil: 2.8.1
>  * social-auth-core: 4.1.0
>  * social-auth-app-django: 4.0.0
>  * django-crispy-forms: 1.11.2
>  * oauthlib: 3.1.0
>  * django-compressor: 2.4.1
>  * djangorestframework: 3.12.4
>  * django-filter: 2.4.0
>  * django-appconf: 1.0.4
>  * user-agents: 2.2.0
>  * filelock: 3.0.12
>  * setuptools: 56.0.0
>  * jellyfish: 0.8.2
>  * openpyxl: 3.0.7
>  * celery: 5.0.5
>  * kombu: 5.0.2
>  * translation-finder: 2.9
>  * weblate-language-data: 2021.4
>  * html2text: 2020.1.16
>  * pycairo: 1.20.0
>  * pygobject: 3.40.1
>  * diff-match-patch: 20200713
>  * requests: 2.25.1
>  * django-redis: 4.12.1
>  * hiredis: 2.0.0
>  * sentry_sdk: 1.0.0
>  * Cython: 0.29.23
>  * misaka: 2.1.1
>  * GitPython: 3.1.14
>  * borgbackup: 1.1.16
>  * pyparsing: 2.4.7
>  * pyahocorasick: 1.4.2
>  * python-redis-lock: 3.7.0
>  * Python: 3.7.3
>  * Git: 2.20.1
>  * psycopg2-binary: 2.8.6
>  * phply: 1.2.5
>  * chardet: 4.0.0
>  * ruamel.yaml: 0.17.4
>  * tesserocr: 2.5.1
>  * akismet: 1.1
>  * boto3: 1.17.53
>  * zeep: 4.0.0
>  * aeidon: 1.9
>  * iniparse: 0.5
>  * Mercurial: 5.7.1
>  * Redis server: 5.0.3
>  * PostgreSQL server: 11.12
>  * Database backends: django.db.backends.postgresql
>  * Cache backends: default:RedisCache, avatar:FileBasedCache
>  * Email setup: django.core.mail.backends.smtp.EmailBackend: localhost
>  * OS encoding: filesystem=utf-8, default=utf-8
>  * Celery: redis://localhost:6379, redis://localhost:6379, regular
>  * Platform: Linux 4.19.0-16-cloud-amd64 (x86_64)

**Comment by ghubz** [comment]
It's like translations have been "Archived" or something. Not all languages are effected and not all translations of the affected languages.

Is there any action that could result in this?

**Comment by nijel** [comment]
"Failed check: Unchanged translation" lists strings whose translation is identical as source. This does not necessarily mean that these are not translated - in some cases it might be a valid translation.

If the strings were really reverted, it should be visible somewhere - either in Weblate history if it was done there or in Git history.
